Events/Prologue.event

```diff
@@ -79,18 +79,20 @@ CHECK_TUTORIAL
 BEQ 0x0 0xC 0x0
 ENUT 0xb0 //turn on casual mode if easy
 LABEL 0x0
-//Text(Ch0IntroTalk)
 LOAD1 0x1 Units1 //0x0 instead of 0x1 seems to be for allies on maps with prep screens
 ENUN
 LOAD1 0x1 EnemyUnits
+CAM1 Vedge
+Text(Prologue_Opening)
 ENUN
```

Comment:
Text before `ENUN`? (Shouldn't matter as this is all done while faded in black, but still)